purchases-sample/src/main/java/com/revenuecat/sample/OfferingCardAdapter.kt

```diff
@@ -19,14 +23,16 @@ class OfferingCardAdapter(private val offerings: List<Offering>, private val lis
     override fun getItemCount(): Int = offerings.size
 
     override fun onBindViewHolder(holder: OfferingViewHolder, position: Int) {
-        holder.bind(offerings[position])
+        val offering = offerings[position]
+        holder.bind(offering, offering == currentOffering)
     }
 
     inner class OfferingViewHolder(private val binding: OfferingCardBinding) :
         RecyclerView.ViewHolder(binding.root) {
 
-        fun bind(offering: Offering) {
+        fun bind(offering: Offering, isCurrent: Boolean) {
             binding.offering = offering
+            binding.isCurrent = isCurrent
```

Comment:
Where's this variable defined? I can't find it in the xml

Reply:
in offering_card.xml!